## Determination of SNP Effect-size of Soybean NAM Population Using Deep Learning

Most of the genome-wide association studies are done using statistical methods where it relies
on the tendency of a genetic marker to inherit along with the causal gene through LD. So far,
statistically driven GWA studies have been shown effective in decoding rare traits and diseases.
In rare phenotypes, the usual scenario is that the SNP association to the phenotype follows a
linear model. However, there are several issues in GWAS for complex traits that undermine the
robustness of the methodology to a great extent. The statistical GWAS consider each SNP
independently for establishing the genotype-phenotype association. Whereas, complex traitsare governed by high-order genetic interactions. Therefore, the statistical methods of GWAS are fundamentally unequipped for the association studies of complex traits. There are many
methods enlisted in the literature capable of performing statistical multi-factor association tests
on complex traits. However, in most polygenic traits the unknown number of interacting SNPs
lays a challenge for them. With the advent of computational superior algorithms and hardware, it
is today possible to carry of studies, such as complex-trait GWAS, where a large number of
parameters are involved. Moreover, statistical GWAS is based on several assumptions about
the distribution of genotypes and their contribution to the phenotype. However, assuming such a
thing is not always practical on the ground. In this study, a deep learning network has been
devised to mitigate the need for prior assumptions about the data and still fit over it to produce
reliable SNP effect sizes.

Read the report for more details.

# Contribution
Not accepting.